lib/matplotlib/projections/polar.py

```diff
@@ -1010,12 +1010,13 @@ def set_thetalim(self, *args, **kwargs):
         """
         Set the minimum and maximum theta values.
 
-        Parameters
-        ----------
-        thetamin : float
-            Minimum value in degrees.
-        thetamax : float
-            Maximum value in degrees.
+        Can take the following signatures:
+
+        - ``set_thetalim(minval, maxval)``: Set the limits in radians.
+        - ``set_thetalim(thetamin=minval, thetamax=maxval)``: Set the limits
+          in degrees.
```

Comment:
I think it would be nice to specify the wrapping around zero behaviour here?

Reply:
Just checked, and wrapping behaviour is pretty wild! Should probably fix it elsewhere before documenting it...
```python
import numpy as np
import matplotlib.pyplot as plt


r = np.arange(0, 2, 0.01)
theta = 2 * np.pi * r

ax = plt.subplot(111, projection='polar')
ax.plot(theta, r)
ax.set_thetalim(0, 3 * np.pi)

plt.show()
```

![Figure_1](https://user-images.githubusercontent.com/6197628/74464110-e0ab5680-4e8a-11ea-9780-6595c2bfee82.png)

Reply:
ha ha, well even just that you can't specify around-zero behaviour with (320, 40)...

Reply:
Ah cool, didn't know you could do that!